app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -69,4 +69,48 @@ def textual_port_config(port_conf)
       :value => "#{port_conf.protocol} port #{port_conf.port} to pods on target port:'#{port_conf.target_port}'"
     }
   end
+
+  def textual_ems
```

Comment:
Remove entirely this method (it has been unified somewhere else so you can just drop it).